cli/shared/src/main/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/Service.scala

```diff
@@ -85,7 +85,7 @@ object Service {
               "kind" -> "Service".asJson,
               "metadata" -> Json(
                 "labels" -> Json(
-                  "appName" -> appName.asJson),
+                  "app" -> appName.asJson),
                 "name" -> appName.asJson)
```

Comment:
Doesn't the `name` need to be `serviceResourceName` in case they differ?

Reply:
I am not sure if the word "service" in `serviceResourceName` means Kubernetes `Service`.
I think the intended purpose is to change the name of the `Deployment` service so when someone changes the version of the app, it would form a new cluster by creating a fresh one.

Reply:
It's used to query the service by name, isn't it?

Reply:
For example when a Lagom service wants to query another Lagom service via DNS?
In that case I'd say you want `appName.asJson` instead of `serviceResourceName` that can potentially be name + version.

Reply:
(In the DNS implementation I mean.)